README.md

```diff
@@ -6,39 +6,58 @@ Java Bindings for the AWS Common Runtime
 
 This library is licensed under the Apache 2.0 License.
 
-## Usage
-### Building
-aws-crt-java uses CMake for setting up build environments. This library depends on:
-* aws-c-common (requires ENABLE_SANITIZERS=OFF, ASan cannot be loaded by JNI currently)
-* aws-c-io
-* s2n (on POSIX platforms)
-* aws-c-mqtt
-* aws-c-http
-
-For example:
-
-    git clone git@github.com:awslabs/aws-crt-java.git aws-crt-java
-    mkdir aws-crt-java-build
-    cd aws-crt-java-build
-    cmake ../aws-crt-java
-    make -j 12
-    make test
-    sudo make install
-
-Keep in mind that CMake supports multiple build systems, so for each platform you can pass your own build system
-as the `-G` option. For example:
-
-    cmake -GNinja ../aws-crt-java
-    ninja build
-    ninja test
-    sudo ninja install
-
-Or on windows,
-
-    cmake -G "Visual Studio 14 2015 Win64" ../aws-crt-java
-    msbuild.exe ALL_BUILD.vcproj
-
-### CMake Options
-* -DCMAKE_CLANG_TIDY=/path/to/clang-tidy (or just clang-tidy or clang-tidy-7.0 if it is in your PATH) - Runs clang-tidy as part of your build.
-* -DCMAKE_INSTALL_PREFIX=/path/to/install - Standard way of installing to a user defined path. If specified when configuring aws-c-common, ensure the same prefix is specified when configuring other aws-c-* SDKs.
-
+## Building
+
+### Linux/Unix
+Requirements: 
+* Clang 3.9+ or GCC 4.4+
+* libssl-dev (on POSIX platforms)
+* cmake 3.1+
+* ninja
+* Java: Any JDK8 or above, ensure JAVA_HOME is set
+* Maven
+
+Building:
+1) apt-get install cmake3 libssl-dev ninja-build maven openjdk-8-jdk-headless -y
+2) git clone https://github.com/awslabs/aws-crt-java.git
+3) cd aws-crt-java
+4) ./build-deps.sh (insert any CMake flags here, such as -DCMAKE_BUILD_TYPE=Debug)
+5) mvn compile
+
+### OSX
+Requirements:
+* cmake 3.1
+* ninja
+* Java: Any JDK8 or above, ensure JAVA_HOME is set
+* Maven
+1) brew install maven cmake3 ninja
```

Comment:
put similar comment to what you did for windows "(if you have homebrew installed, otherwise..."